### ISSUE TITLE
union alignment computation seems wrong

### ISSUE THREAD

**Issue by mwette** [TaylanUB/scheme-bytestructures#27]
The following code yields size=43, alignment=43.  Shouldn't alignment be 8?
```
(use-modules (bytestructures guile))
(define u (bs:union `((x ,double) (y ,(bs:vector 43 uint8)))))
(simple-format #t "siz=~S\n" (bytestructure-descriptor-size u))
(simple-format #t "aln=~S\n" (bytestructure-descriptor-alignment u))
```

**Comment by amirouche** [comment]
There was fix related to this in 1.0.1. Are you using that version?

**Comment by mwette** [comment]
I thought so, but I just performed a git pull and retried.  Same problem.
Then I removed the cached .go files and it now works.   Thanks!
